scripts/obtain/dotnet-install.sh

```diff
@@ -104,11 +108,15 @@ get_current_os_name() {
                     echo "ubuntu.16.10"
                     return 0
                     ;;
+                "alpine.3.4.3")
+                    echo "alpine"
```

Comment:
Right now it is just a bootstrap package and this is the name we gave it, since we don't yet have real builds coming out for it with real versioning. When we get stuff all the way through to CLI, we can update it as necessary.